Test/test2/Rlimitouts0.bpl

```diff
@@ -0,0 +1,87 @@
+// RUN: %boogie -rlimit:8000 "%s" > "%t"
+// RUN: %diff "%s.expect" "%t"
```

Comment:
Seeing as you're adding a new test you should probably use `%OutputCheck` rather than the incredibly fragile `%diff`.